src/fungible_token.rs

```diff
@@ -421,7 +421,20 @@ impl FungibleToken {
     }
 
     pub fn accounts_insert(&self, account_id: &str, amount: Balance) {
-        sdk::save_contract(&Self::account_to_key(account_id), &amount)
+        if self.accounts_contains_key(account_id) {
+            let key = Self::get_statistic_key();
+            let accounts_counter = sdk::read_u64(&key)
+                .unwrap_or(0)
+                .checked_add(1)
+                .expect("ERR_ACCOUNTS_COUNTER_OVERFLOW");
+            sdk::write_storage(&key, &accounts_counter.to_le_bytes());
+        }
+        sdk::save_contract(&Self::account_to_key(account_id), &amount);
+    }
+
+    /// Get accounts counter for statistics
```

Comment:
I'd appreciate a little more detail in this comment. What does this counter actually measure? Is it the all-time number of distinct accounts? The number of accounts currently holding a non-zero balance? Something else?

I know someone could read the code to figure out what exactly the counter counts, but I think a high level understanding should be available without getting into the weeds of the code.

Reply:
Total unique accounts. I'll extend that comment and PR description.

Reply:
Done